ocamldoc/odoc_html.ml

```diff
@@ -498,7 +498,7 @@ class virtual text =
     method virtual html_of_info_first_sentence : _
 
     method html_of_Module_list b l =
-      bs b "<br>\n<table class=\"indextable\">\n";
+      bs b "<div class=\"module-list\">\n<table class=\"indextable\">\n";
```

Comment:
Rather than add an enclosing `div` I would simply add `module-list` to the `class` list.